iCount/files/bedgraph.py

```diff
@@ -0,0 +1,59 @@
+""".. Line to protect from pydocstyle D205, D400.
+
+Bedgraph conversion
+-------------------
+
+Convert from BED6 to bedGraph format.
+
+"""
+import logging
+
+import pybedtools
+
+import iCount
+
+LOGGER = logging.getLogger(__name__)
+
+
+def bed2bedgraph(bed, bedgraph, name='User Track', description='User Supplied Track'):
+    """
+    Convert from BED6 to bedGraph format.
+
+    For further explanation of ``name`` and ``description`` parameters
+    (and also others, that are not yet supported) see:
+    https://genome.ucsc.edu/goldenPath/help/customTrack.html#TRACK
+
+    Parameters
+    ----------
+    bed : str
+        Input BED6 file.
+    bedgraph : str
+        Output bedGraph file.
+    name : str
+        Track label. Can consist of up to 15 characters.
+    description : str
+        Track description. Can consist of up to 60 characters.
+
+    Returns
+    -------
+    None
+
+    """
+    iCount.log_inputs(LOGGER, level=logging.INFO)  # pylint: disable=protected-access
+    LOGGER.info('Checking input parameters...')
+    assert bed.endswith(('.bed', '.bed.gz'))
+
+    header = [
+        'track',
+        'type=bedGraph',
+        'name="{}"'.format(name[:15]),
+        'description="{}"'.format(description[:60]),
+    ]
+
+    LOGGER.info('Converting %s to %s', bed, bedgraph)
+    with open(bedgraph, 'wt') as outfile:
+        print(' '.join(map(str, header)), file=outfile)
```

Comment:
@JureZmrzlikar: why use `print` instead of `.write` to write to files?

Reply:
Why not? Isn't this is a matter of preference?
https://stackoverflow.com/questions/8598228/f-write-vs-print-f

Reply:
For consistency sake. Everywhere else we use `.write`. Can you please use it here as well.

Reply:
Of course, no problem.